tests/test_host_info_plugin.py

```diff
@@ -62,6 +63,32 @@ def host_data(mocker):
 
 
 @pytest.fixture()
+def no_cadvisor_host_data(mocker):
+    mocker.patch.object(platform, 'system', return_value='Linux')
+    mocker.patch('os.getloadavg',
+                 return_value=(1.60693359375, 1.73193359375, 1.79248046875))
+
+    mocker.patch.object(CpuCollector,
+                        '_get_cpuinfo_data',
+                        return_value=cpuinfo_data())
+
+    mocker.patch.object(MemoryCollector,
+                        '_get_meminfo_data',
+                        return_value=meminfo_data())
+
+    mocker.patch.object(CadvisorAPIClient, '_get',
+                        return_value=None)
+
+    mocker.patch('cattle.utils.check_output',
```

Comment:
Thats a utility that runs arbitrary commands. I'm just testing that the method gets called with the correct args.. not so much the value. It happened to be the actual value I expected at the time I wrote the test a while ago.